src/Game.js

```diff
@@ -0,0 +1,78 @@
+function Game() {
+  this.frames = [[[]]];
```

Comment:
Yes, the code might be clearer if a Frame object was used to model the individual frames.